google-cloud-bigtable/lib/google/cloud/bigtable/row_filter/chain_filter.rb

```diff
@@ -151,15 +151,15 @@ def interleave filter
           # @example
           #   require "google/cloud/bigtable"
           #
-          #   predicate = Google::Cloud::Bigtable::RowFilter.key("user-*")
+          #   predicate = Google::Cloud::Bigtable::RowFilter.key "user-*"
           #
-          #   label = Google::Cloud::Bigtable::RowFilter.label("user")
+          #   label = Google::Cloud::Bigtable::RowFilter.label "user"
           #   strip_value = Google::Cloud::Bigtable::RowFilter.strip_value
           #
-          #   condition_filter = Google::Cloud::Bigtable::RowFilter.
-          #     condition(predicate).on_match(label).otherwise(strip_value)
+          #   condition_filter = Google::Cloud::Bigtable::RowFilter.condition(predicate)
+          #                                                        .on_match(label).otherwise(strip_value)
```

Comment:
I would put `.otherwise` on the next line.

Reply:
Done. Thanks!